src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Illuminate\Database\Eloquent;
 
+use Lang;
```

Comment:
Hey, thanks for the feedback - I've just adjusted that to the FQCN of the Facade.